packages/dx-demo-shell/generate-demos.js

```diff
@@ -183,7 +184,13 @@ const groupBy = (array, iteratee) => array
     return acc;
   }, {});
 const indent = (string, count) => string.split('\n').map(substring => `${' '.repeat(count)}${substring}`).join('\n');
-const generateDemoRegistry = () => {
+const getCurrentProductName = () => {
+  const currentPath = path.resolve('./').split('/');
+  const packageName = currentPath[currentPath.length - 1];
```

Comment:
Isn't `process.cwd` more straightforward then `path.resolve('./')`?
And there is `path.basename`.

```javascript
const packageName = path.basename(process.cwd());
```